docs/commands.md

```diff
@@ -408,6 +408,10 @@ Starts the server that communicates with connected devices
 
 Specify port to listen on
 
+#### `--projectRoot [list]`
```

Comment:
this isn't actually a list is it? looks like it's just forwarded to `path.resolve()` which doesn't take a list

Reply:
Good point, it's a string. Mind sending a PR?

Reply:
https://github.com/react-native-community/cli/pull/632